CHANGELOG.md

```diff
@@ -55,6 +55,9 @@ and this project adheres to
   - [#4126](https://github.com/bpftrace/bpftrace/pull/4126)
 - For loops now support `break` and `continue`
   - [#4250](https://github.com/bpftrace/bpftrace/pull/4250)
+- Add `pid` and `tid` functions for choosing between the initial or the current
+  namespace
```

Comment:
super nit: let's keep this on the same line

Reply:
Done.